### PR TITLE
improve limel-tooltip

### PR DESCRIPTION
**Pull request by Kiarokh**
Improving the docs and examples, as well as minor CSS tweaks.

## Review:
- [x] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [x] Commits have the correct *type* for the changes made
- [x] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [x] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/design-guidelines/declutter/tooltip-declutter.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                    <limel-header icon="ok" heading="Do"></limel-header>
+                    <div class="fake-dialog-container shows-full-dialog">
+                        <div class="fake-dialog">
+                            <p>
+                                Schedule this task?{' '}
+                                <span
+                                    class="tooltip-trigger"
+                                    id="tooltip-example"
+                                >
+                                    More…
```
[the remark this answers is not in the document]
Comment (Kiarokh) [reply src/design-guidelines/declutter/tooltip-declutter.tsx]: You're 1245% right! I'll fine-tune the design.

File: src/components/tooltip/tooltip.tsx [hunk 2]
```diff
@@ -2,17 +2,43 @@ import { Component, h, Prop, Element, State } from '@stencil/core';
...
+ * - Keep in mind that tooltips can be precieved as interruption and distracting.
+ * Use them only when they add significant value.
+ * - A good tip is concise, helpful, and informative.
+ * Don't explain the obvious or simply repeat what is already on the screen.
+ * When used correctly, supplemental info of a tooltip helps to [declutter the UI](/#/DesignGuidelines/decluttering.md/).
+ * - If the tip is essential to the primary tasks that user should do in the UI,
+ * such as warnings or important notes, include the information directly in the
+ * interface instead.
+ * - When a component offers a helper text (e.g. [Input field](/#/component/limel-input-field/)),
+ * use that not a tooltip.
```
[the remark this answers is not in the document]
Comment (Kiarokh) [reply src/components/tooltip/tooltip.tsx:39]: 😆

File: src/design-guidelines/declutter/tooltip-declutter.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (adrianschmidt) [reply src/design-guidelines/declutter/tooltip-declutter.tsx]: # 🤣 ❤️

[the remark this answers is not in the document]
Comment (Kiarokh) [reply src/design-guidelines/declutter/tooltip-declutter.tsx]: done

[the remark this answers is not in the document]
Comment (adrianschmidt) [reply src/design-guidelines/declutter/tooltip-declutter.tsx]: Yeah, that's a lot better! Good job! 👏

[the remark this answers is not in the document]
Comment (Kiarokh) [reply src/design-guidelines/declutter/tooltip-declutter.tsx]: Thanks! Did I meet all of your requirements ? 😄 Or anything left?

[the remark this answers is not in the document]
Comment (adrianschmidt) [reply src/design-guidelines/declutter/tooltip-declutter.tsx]: Oh, sorry, I got distracted in reporting a bug in a browser plugin I'm using… sorry about that! 😅

**Review by adrianschmidt — APPROVED** [review]

**Pull request merged**